src/index.js

```diff
@@ -44,4 +42,26 @@ for (let i = 0; i < deleteIcon.length; i += 1) {
     dots[i].classList.remove('hidden');
     trash[i].classList.add('hidden');
   });
-}
\ No newline at end of file
+  trash[i].addEventListener('click', () => {
+    tasks.splice(i, 1);
+    localStorage.setItem('newItem', JSON.stringify(tasks));
+    window.location.reload();
+  });
+  checkbox[i].addEventListener('input', () => {
+    if (tasks[i].completed) {
+      tasks[i].completed = false;
+      currentTask[i].classList.remove('strike');
+    } else {
+      tasks[i].completed = true;
+      currentTask[i].classList.add('strike');
+    }
+    localStorage.setItem('newItem', JSON.stringify(tasks));
+  });
+  currentTask[i].addEventListener('keydown', (e) => {
+    if (e.code === 'Enter') {
+      tasks[i].description = currentTask[i].innerHTML;
+      localStorage.setItem('newItem', JSON.stringify(tasks));
+      window.location.reload();
+    }
+  });
```

Comment:
- [ ] Please make sure deleting a task should update all remaining items' indexes/ids, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1) as requested in the project requirements. Currently deleting a task doesn't update the unique indexes/ids of the tasks.
  - As in the below image you can see I deleted the first task from the list, and the index/ids of the remaining tasks are not updated and this is causing the duplication of id. Please fix this issue.
![image](https://user-images.githubusercontent.com/83803414/220698444-d6519e25-82cf-4ecd-adbd-6749694f86bd.png)

Reply:
@thecodechaser thanks for your kind review.
actually i spent much time to develop this functionality but couldn't. If you can please help me in it.

Reply:
@DilsherB I would suggest after remove event
Start a loop for the list array and update the ids of all items in the list with i+1